controllers/authentication/openidconnect_controller.go

```diff
@@ -308,3 +357,22 @@ func unmarshalResp(r *http.Response, body []byte, v interface{}) error {
 	}
 	return fmt.Errorf("expected Content-Type = application/json, got %q: %v", ct, err)
 }
+
+// loadKey parses the jwks key Set, extracts public rsa keys found and converts them into a JOSE JWK format.
+func loadKey(jwks []byte) ([]*jose.JSONWebKey, error) {
+	var keyList []*jose.JSONWebKey
+	pubKeyJwk, err := jwk.ParseString(string(jwks))
```

Comment:
`json.Unmarshal` instead?

Reply:
yes could do, but github.com/lestrrat-go/jwx/jwk simplifies realising the rsa Public key

Reply:
Keep it as much as possible to `coreos/go-oidc` 

See https://github.com/coreos/go-oidc/blob/15b94d97d90c9455981dbdd90a021d34254cfdaf/oidc/jwks.go#L181-L208
and https://github.com/coreos/go-oidc/blob/15b94d97d90c9455981dbdd90a021d34254cfdaf/oidc/oidc.go#L442-L453

Reply:
done